src/main/resources/swagger/api-docs.yaml

```diff
@@ -295,3 +484,80 @@ definitions:
       - methodConfigurationName
       - entityType
       - entityName
+  ConfigurationIngest:
+    properties:
+      namespace:
+        type: string
+        description: Method configuration namespace
+      name:
+        type: string
+        description: Method configuration name
+      rootEntityType:
+        type: string
+        description: Root entity type
+      workspaceName:
+        type: string
```

Comment:
Swagger supports it at least sort of - see https://rawls-dev.broadinstitute.org/api-docs/listings/methodconfigs (or the rawls swagger UI).

I just tried
```
type: Map[string,string]
```
locally. That makes it appear correctly in the "Model" view, but makes it disappear from the "Model Schema" view. Hmmmmm.